### PR TITLE
chore(interview): rebuild NarrativePedigree storybook around one realistic family

### PR DESCRIPTION
**Pull request by jthrilly**
## What

Reworks the `NarrativePedigree` storybook so the interface is demonstrated by **one realistic family** instead of a handful of small fixtures.

- **One args-driven `Default` story** replaces the seven `Interfaces/NarrativePedigree` stories and the separate `AllInheritancePatterns` story. The `showAtRiskStatuses` control (default **on**) toggles the probabilistic markers.
- **`comprehensivePedigreeFixture` is reseeded** as a single integrated five-generation family. Each of the six conditions in the condition key gets its own genetically realistic branch, and — the point of the change — **every notation symbol now appears**, including the two the old fixture never surfaced in the shown view: *"will develop this condition"* (obligate-affected) and *"two copies"* (at-risk-homozygous).
- **Realistic North-American names** with patrilineal surname inheritance (the two CF cousins even carry different surnames because one descends through a son, the other a daughter of the shared grandparents).
- **One egg-donation branch**: an at-risk mother's donor-conceived child inherits from the donor, not from her, so she escapes the family's maternal-line risk — a `donor` (genetic) + `social`/gestational (non-genetic) edge pairing.

## How each condition manifests (all engine-verified)

| Condition | Pattern | In the tree |
|---|---|---|
| Huntington's | AD | Arthur → Rose affected; the maternal descent at risk |
| Mitochondrial myopathy | mt | Matriline from Eleanor; at-risk males don't transmit, females do |
| Y-linked hearing loss | Y | Sullivan male line Harold→David→Ben; Owen inferred "will develop it" |
| Haemophilia A | XLR | Two affected brothers → obligate-carrier mother → carrier females to ego & daughter |
| X-linked hypophosphataemia | XLD | Walter's daughters all affected, his son (and ego's children) spared |
| Cystic fibrosis | AR | Consanguineous first-cousin union → affected child + "two copies" sibling |

## Scope

`Examples/CEGRM`, `Examples/Pedigree Flow`, and the component-level stories are left in place (Pedigree Flow just picks up the richer family). The `Capture` story is repointed and the `NarrativePedigree.*` interface images are regenerated (drift on the other interfaces was restored, so the diff is NarrativePedigree-only).

## Test plan

- `geneticRealism.test.ts` runs the real genetics engine on the fixture and asserts each condition's pattern, full notation-symbol coverage, the egg-donation escape, and the consanguinity wiring.
- Determinism guard renamed to `comprehensivePedigreeFixture.test.ts`; redundant `allPathways.test.ts` removed.
- `pnpm --filter @codaco/interview typecheck` ✅ · `pnpm knip` ✅ · lint/format ✅ · full interview unit suite (124 files / 1124 tests) ✅
- Rendered the `Default` story in Storybook and confirmed the 31-node pedigree, consanguinity loop, egg-donation branch, status stickers, and the "two copies" marker all render.

No changeset: storybook-fixture + test only (no consumer-visible runtime change); interface-images is a private package.

Design note: `docs/superpowers/specs/2026-07-07-narrative-pedigree-storybook-design.md`.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **New Features**
  * Added a richer multi-generation pedigree example with broader inheritance coverage, including an egg-donation branch and more status symbols.
  * Simplified the NarrativePedigree Storybook to a single, configurable default story with an at-risk display toggle.

* **Bug Fixes**
  * Improved pedigree consistency for genetic relationships and edge cases across inheritance patterns.

* **Tests**
  * Expanded automated checks to verify the new comprehensive pedigree remains stable and genetically coherent.

### CODE REVIEW

File: packages/interview/src/interfaces/NarrativePedigree/NarrativePedigree.capture.stories.tsx [hunk 1]
```diff
@@ -4,20 +4,19 @@ import CaptureStory, {
...
- * capture lands on the NarrativePedigree stage itself (step index 1; the
- * FamilyPedigree source is stage 0). The multi-disease network renders the
- * sticker view, picturing the genetics overlay.
+ * pedigree collected by a FamilyPedigree source stage, so the build seeds the
+ * populated multi-disease pedigree and the capture lands on the NarrativePedigree
+ * stage itself (step index 1; the FamilyPedigree source is stage 0). The
+ * multi-disease network renders the sticker view, picturing the genetics overlay.
  */
-const build = () => buildPedigreeInterview(1);
+const build = () => buildComprehensivePedigree(1);
```
[the remark this answers is not in the document]
Comment (jthrilly) [reply packages/interview/src/interfaces/NarrativePedigree/NarrativePedigree.capture.stories.tsx]: Fixed — the capture story now passes `showAtRisk` explicitly (`buildComprehensivePedigree(1, true)`), pinning the generated image to the at-risk overlay regardless of the builder default. (74773b61c)

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: packages/interview/src/interfaces/NarrativePedigree/__tests__/geneticRealism.test.ts [hunk 2]
```diff
@@ -2,117 +2,190 @@ import { describe, expect, it } from 'vitest';
...
+  const homozygousFor = (variable: string, pattern: InheritancePattern) => {
+    const statuses = statusesFor(variable, pattern);
+    return computeAtRiskHomozygous(graph, statuses, pattern, resolveSexFn);
   };
+  const statusOf =
+    (variable: string, pattern: InheritancePattern) =>
+    (uid: string): Status =>
+      statusesFor(variable, pattern).get(uid) ?? 'unknown';
+  return { nodes, graph, statusesFor, homozygousFor, statusOf };
 }
```
[the remark this answers is not in the document]
Comment (jthrilly) [reply packages/interview/src/interfaces/NarrativePedigree/__tests__/geneticRealism.test.ts]: Done — hoisted to a module-level `const engine = buildEngine()`. Safe to share: the fixture is seed-deterministic and `computeStatuses`/`computeAtRiskHomozygous` return fresh Maps and never mutate the graph, so there is no cross-test contamination.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: packages/interview/src/interfaces/NarrativePedigree/__tests__/geneticRealism.test.ts [hunk 3]
```diff
@@ -2,117 +2,190 @@ import { describe, expect, it } from 'vitest';
...
+
+function engine() {
+  const { nodes, edges } = buildComprehensivePedigree(1).getNetwork();
   const resolveSexFn = (id: string) => resolveSex(id, nodes, edges, CFG);
   const graph = buildGeneticGraph(
     nodes,
     edges,
     { relationshipTypeVariable: 'relType' },
     resolveSexFn,
   );
```
[the remark this answers is not in the document]
Comment (jthrilly) [reply packages/interview/src/interfaces/NarrativePedigree/__tests__/geneticRealism.test.ts]: Done — `buildGeneticGraph` now takes the shared `CFG` (its `GeneticGraphConfig` param only needs `relationshipTypeVariable`, which `CFG` already carries), removing the duplicated `'relType'` literal.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
